prisma/schema.prisma

```diff
@@ -610,6 +610,7 @@ enum DaoSlug {
   OP
   ENS
   UNI
+  ETHFI
   NOUNS
   LYRA
   ETHERFI
```

Comment:
This `ETHERFI` slug reflects the DB state. I can update to `ETHFI` on the DB if you think it's a better name

Reply:
Good catch, I will remove that. Their token is ETHFI - i think ETHERFI makes more sense as a namespace.